src/components/Calculator.js

```diff
@@ -29,78 +74,135 @@ class Calculator extends Component {
               </Button>
             </Col>
           </Row>
           <Row>
             <Col xs={12} className="number-container">
-              <Button className="button numkb">
+              <Button
+                className="button numkb"
+                onClick={() => this.buttonPress('4')}
+              >
                 4
               </Button>
-              <Button className="button numkb">
+              <Button
+                className="button numkb"
+                onClick={() => this.buttonPress('5')}
+              >
                 5
               </Button>
-              <Button className="button numkb">
+              <Button
+                className="button numkb"
+                onClick={() => this.buttonPress('6')}
+              >
                 6
               </Button>
-              <Button className="button symbols">
-                -
+              <Button
+                className="button symbols"
+                onClick={() => this.buttonPress('-')}
+              >
+                <FontAwesomeIcon icon={faMinus} />
               </Button>
             </Col>
           </Row>
           <Row>
             <Col xs={12} className="number-container">
-              <Button className="button numkb">
+              <Button
+                className="button numkb"
+                onClick={() => this.buttonPress('1')}
+              >
                 1
               </Button>
-              <Button className="button numkb">
+              <Button
+                className="button numkb"
+                onClick={() => this.buttonPress('2')}
+              >
                 2
               </Button>
-              <Button className="button numkb">
+              <Button
+                className="button numkb"
+                onClick={() => this.buttonPress('3')}
+              >
                 3
               </Button>
-              <Button className="button symbols">
-                +
+              <Button
+                className="button symbols"
+                onClick={() => this.buttonPress('+')}
+              >
+                <FontAwesomeIcon icon={faPlus} />
               </Button>
             </Col>
           </Row>
           <Row>
             <Col xs={12} className="number-container">
-              <Button className="button btn-zero numkb">
+              <Button
+                className="button btn-zero numkb"
+                onClick={() => this.buttonPress('0')}
+              >
                 0
               </Button>
-              <Button className="button numkb">
+              <Button
+                className="button numkb"
+                onClick={() => this.buttonPress('.')}
+              >
                 .
               </Button>
-              <Button className="button symbols">
-                =
+              <Button
+                className="button symbols"
+                onClick={() => this.buttonPress('=')}
+              >
```

Comment:
- [ ] I think you can move all of this code to a new component ("CalculatorButtons") to avoid all of the unrelated code to the functionality of the calc. Also, you will practice how to compose components and how to pass props down.

Reply:
this requirement isn't in the project requirements where i can found more information related to it? @D4nielJ

Reply:
@D4nielJ

Reply:
@D4nielJ

Reply:
[You can read more about it here](https://reactjs.org/docs/components-and-props.html).

Reply:
** @D4nielJ  I'm not using it?** 
> Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift state up.

Reply:
Well, you are not passing props down since you built all of the components on the same file. I think knowing how to lift state and pass props down is a valuable skill when developing on react and also the point of this lesson.

Also, I reviewed the changes you made and I don't agree with them. You moved the entire component one level down, instead, you could try just moving the buttons from the calculator, and leaving the logic on the Calculator component.

Reply:
try to explain better which changes you want! :)

Reply:
I'm sorry my explanation was not good enough. I will definitely take this into account for future reviews. Happy coding :sweat_smile: